shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -48,6 +48,7 @@
     public static final String CONTAINER_CREATE_TXN_COUNT = "container_create_txn_count";
     public static final String CONTAINER_MERGE_TXN_COUNT = "container_merge_txn_count";
     public static final String CONTAINER_SEAL_COUNT = "container_seal_count";
+    public static final String CONTAINER_TRUNCATE_COUNT = "container_truncate_count";
```

Comment:
Is this the number of truncate operations?

Reply:
yes, similar to the ones above it.